### PR TITLE
Use a real softmax op

### PR DESCRIPTION
**Pull request by blester125**
In the old model a new `tf.nn.softmax` was created for each call to `classify`. This PR created a dedicated softmax op and runs with that instead.

A quick test running through the SST2 dev set calling `classify_text` took 80s with the old way. The new way takes 3s

### CODE REVIEW

File: python/baseline/tf/classify/model.py [hunk 1]
```diff
@@ -403,6 +404,7 @@ def create(cls, embeddings, labels, **kwargs):
                 with tf.variable_scope("output"):
                     model.logits = tf.identity(fully_connected(stacked, nc, activation_fn=None), name="logits")
                     model.best = tf.argmax(model.logits, 1, name="best")
+                    model.probs = tf.nn.softmax(model.logits, name="probs")
```
Comment (dpressel) [inline python/baseline/tf/classify/model.py]: For backwards compat, I would think that `load()` needs to be updated to create this if its not found

Comment (blester125) [reply python/baseline/tf/classify/model.py]: Fixed. tested by training a model on master and reloading it in this branch
